dune-project

```diff
@@ -0,0 +1,5 @@
+(lang dune 2.0)
+
+(name vg_canvas)
+
+(formatting disabled)
```

Comment:
Is there a reason to disable this?  My instinct would be to just turn it on.

Reply:
Ah I guess the auto-converter put it there for some reason. Removed.